Tests/TestingTests/RunnerTests.swift

```diff
@@ -332,6 +332,23 @@ final class RunnerTests: XCTestCase {
     let test2 = Test(.disabled(if: Bool.random())) { }
     XCTAssertTrue(test2.traits.compactMap { $0 as? ConditionTrait }.allSatisfy { !$0.isConstant })
   }
+  
+  func testEvaluateConditionTrait() async throws {
```

Comment:
We now have a test file named `ConditionTraitTests`, whose tests are implemented using Swift Testing instead of XCTest which this file still uses for legacy reasons. I think this test would make more sense in that new file.

Reply:
I had to add `@_spi(Experimental)` to the import in that file, which seems like it should be fine.